pkgs/development/tools/rust/cargo-pretty-test/default.nix

```
@@ -0,0 +1,21 @@
+{lib, fetchgit, stdenv, fetchFromGitHub, rustPlatform}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-pretty-test";
+  version = "v0.2.3";
```

This is a tiny thing but:
```suggestion
  version = "0.2.3";
```

See [here](https://github.com/NixOS/nixpkgs/pull/256548#discussion_r1336761932) for source. I couldn't find a reference in the documentation.